### ISSUE TITLE
Request training code for transfering to detection and segmentation

### ISSUE THREAD

**Issue by mZhenz** [jacobswan1/SEED#8]
Hi, I am using your released pretrained checkpoint and trying to transfer it to detection and segmentation task on COCO, but I can get the same result as that in your paper. I am wondering if you could kindly release your training code for this part, or just simply release your experiment's config files of the detectron2. Thanks a lot!

**Comment by jacobswan1** [comment]
Hi Zhen, 

Thanks for the suggestion. 

I didn't organize the previous detection implementations but I think these settings might be helpful for the detection task: 

Reduce the IMS_PER_BATCH to 8, BASE_LR to 0.01, and enlarging the converging epochs 2X is helpful for the results. It would also be good if you could first try out Kaiming's MoCo detection settings and see if the correct results can be reproduced using the exact settings: https://github.com/facebookresearch/moco/tree/main/detection using ResNet50.

For the specific config files, I found these settings that might be helpful:

coco_R_18_C4_2x.yaml
```
_BASE_: "Base-RCNN-C4-BN.yaml"
MODEL:
  MASK_ON: True
  WEIGHTS: "detectron2://ImageNetPretrained/MSRA/R-18.pkl"
INPUT:
  MIN_SIZE_TRAIN: (640, 672, 704, 736, 768, 800)
  MIN_SIZE_TEST: 800
DATASETS:
  TRAIN: ("coco_2017_train",)
  TEST: ("coco_2017_val",)
SOLVER:
  STEPS: (240000, 320000)
  MAX_ITER: 360000
```

Base-RCNN-C4-BN.yaml
```
MODEL:
  META_ARCHITECTURE: "GeneralizedRCNN"
  RPN:
    PRE_NMS_TOPK_TEST: 6000
    POST_NMS_TOPK_TEST: 1000
  ROI_HEADS:
    NAME: "Res5ROIHeadsExtraNorm"
  BACKBONE:
    FREEZE_AT: 0
  RESNETS:
    NORM: "SyncBN"
TEST:
  PRECISE_BN:
    ENABLED: True
SOLVER:
  IMS_PER_BATCH: 8
  BASE_LR: 0.01
```

coco_R_18_C4_2x_moco.yaml
```
_BASE_: "coco_R_18_C4_2x.yaml"
MODEL:
  PIXEL_MEAN: [123.675, 116.280, 103.530]
  PIXEL_STD: [58.395, 57.120, 57.375]
  WEIGHTS: "See Instructions"
  RESNETS:
    DEPTH: 18
    STRIDE_IN_1X1: False
    RES2_OUT_CHANNELS: 64
INPUT:
  FORMAT: "RGB"
```

**Comment by mZhenz** [comment]
@jacobswan1 Hi, thank you so much for offering the training config files. I try to use it but I still can't get the same results as that reported in your paper and there is a large gap (3% for AP). The [config file](https://github.com/mZhenz/Work-Tencent/blob/main/config.yaml) and [training log](https://github.com/mZhenz/Work-Tencent/blob/main/log.txt) are here. The pretrained weights I used is the resnet18_distill_resnet50-moco-v2-checkpoint_0199.pth.tar released in this repo. I am so confused and it would be nice to have your advice.

**Comment by mZhenz** [comment]
If possible, could you offering the training log of this experiment? Thanks a lot.
